examples/kubernetes/mlflow/k8s-manifest.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-secrets
+type: Opaque
+stringData:
+  POSTGRES_PASSWORD: "superSecretPassword"
+  POSTGRES_USER: "optuna"
+  POSTGRES_DB: "optunaDatabase"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
+spec:
+  type: ClusterIP
+  selector:
+    app: postgres
+  ports:
+    - port: 5432
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgres
+  labels:
+    app: postgres
+spec:
+  selector:
+    matchLabels:
+      app: postgres
+  serviceName: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres:latest
+          imagePullPolicy: IfNotPresent
+          envFrom:
+            - secretRef:
+                name: postgres-secrets
+          ports:
+            - containerPort: 5432
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mlflow
+spec:
+  type: ClusterIP
```

Comment:
Not a strong opinion but you can also fix the node port. A user could then just obtain the cluster ip (e.g. `minikube ip`) and append this port to open the dashboard without having to check the generate port (e.g. using `minikube service mlflow --url`).